i18n/en.toml

```diff
@@ -1,11 +1,5 @@
 # i18n strings for the English (main) site.
 # NOTE: Please keep the entries in alphabetical order when editing
-[announcement_title]
-other = "Black lives matter."
-
```

Comment:
nit: Could add this announcement data into the announcement template.

Reply:
see discussion of current announcement data in https://github.com/kubernetes/website/pull/25769#issuecomment-756121732